src/java/htsjdk/samtools/util/zip/ZStreamRef.java

```diff
@@ -1,48 +0,0 @@
-/*
```

Comment:
Is this no longer needed because of Java 8? Otherwise the IntelDeflator needs this.